action.yaml

```diff
@@ -83,8 +88,25 @@ runs:
         any::pkgdepends
         any::cli
         any::desc
+        any::pkgcache
       install-pandoc: false
 
+  - name: Set repositories
+    run: |
+      cat > ${HOME}/.Rprofile <<EOF
+      repository_list <- unlist(strsplit(
+        "${{ inputs.repository-list }}", split=","
+      ))
+      options(repos = c(vapply(repository_list, pkgcache::repo_resolve, character(1)), getOption("repos")))
+      EOF
+      R -s -q <<RSCRIPT
+      cat("Resolved repositories loaded from .Rprofile:\n")
+      getOption("repos")
+      RSCRIPT
```

Comment:
following https://github.com/insightsengineering/setup-r-dependencies/pull/8#pullrequestreview-2328628600
do we need this?